spyder/plugins/debugger/plugin.py

```diff
@@ -15,14 +18,18 @@
 from spyder.plugins.debugger.widgets.main_widget import (
     DebuggerWidget)
 from spyder.api.shellconnect.mixins import ShellConnectMixin
+from spyder.utils.qthelpers import MENU_SEPARATOR
+from spyder.config.manager import CONF
+from spyder.plugins.mainmenu.api import ApplicationMenus
+from spyder.plugins.debugger.widgets.main_widget import DebuggerWidgetActions
```

Comment:
```suggestion
from spyder.plugins.debugger.widgets.main_widget import DebuggerToolbarActions
```

I think the `Debug file` and `Debug cell` actions should be part of a different enum class.